tests/suites/test_suite_psa_crypto_metadata.function

```diff
@@ -457,3 +457,22 @@ void ecc_key_types( int curve_arg, int curve_bits_arg )
     TEST_ASSERT( curve_bits <= PSA_VENDOR_ECC_MAX_CURVE_BITS );
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_DHM_C */
+void dh_key_types( int group_arg, int group_bits_arg )
+{
+    psa_dh_group_t group = group_arg;
+    size_t group_bits = group_bits_arg;
+    psa_key_type_t public_type = PSA_KEY_TYPE_ECC_PUBLIC_KEY( group );
```

Comment:
Copypasta: this should be `PSA_KEY_TYPE_DH_PUBLIC_KEY`. Same goes for the macros in the lines below.

Reply:
I made a fixup commit to fix this and add `KEY_TYPE_IS_DH`.